packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -379,9 +379,10 @@ protected void onDraw(Canvas canvas) {
             getIncludeFontPadding(),
             getBreakStrategy(),
             getHyphenationFrequency(),
-            // always passing ALIGN_NORMAL here should be fine, since this method doesn't depend on
-            // how exacly lines are aligned, just their width
+            // always passing ALIGN_NORMAL and JUSTIFICATION_MODE_NONE here should be fine, since this method doesn't depend on
+            // how exactly lines are aligned, just their width
```

Comment:
I don't think this is correct, since justification can change line width.

Reply:
Yeah we could get current justification mode or pass -1 as well I think.